lib/platform-stats.js

```diff
@@ -0,0 +1,26 @@
+import createDebug from 'debug'
+
+const debug = createDebug('spark:platform-stats')
+
+/**
+ * @param {import('pg').Client} pgClient
+ * @param {import('./preprocess').Measurement[]} honestMeasurements
+ */
+export const updatePlatformStats = async (pgClient, honestMeasurements) => {
+  await updateDailyNodeMetrics(pgClient, honestMeasurements)
+}
+
+/**
+ * @param {import('pg').Client} pgClient
+ * @param {import('./preprocess').Measurement[]} honestMeasurements
+ */
+export const updateDailyNodeMetrics = async (pgClient, honestMeasurements) => {
+  debug('Updating daily node metrics, count=%s', honestMeasurements.length)
+  for (const m of honestMeasurements) {
+    await pgClient.query(`
+      INSERT INTO daily_node_metrics (station_id, day)
+      VALUES ($1, now())
+      ON CONFLICT (station_id, day) DO NOTHING
+    `, [m.stationId]) // TODO: when we add more fields, we should update the ON CONFLICT clause to update the fields
+  }
```

Comment:
We have thousands of participants in each round, this code will run thousands of SQL queries and will take very long to complete. I have been bitten by this in the past and had to use a different solution:

https://github.com/filecoin-station/spark-evaluate/blob/562125a2c1bc99b4aaf0fceba4c4bb0a872913b5/lib/public-stats.js#L68-L76

Can you do the same?